app/src/main/java/com/tehmou/rxbookapp/utils/TextWatcherObservable.java

```diff
@@ -0,0 +1,48 @@
+package com.tehmou.rxbookapp.utils;
```

Comment:
Why not using RxAndroid stuff? https://github.com/ReactiveX/RxAndroid/blob/0.x/rxandroid/src/main/java/rx/android/widget/OnTextChangeEvent.java